spec/requests/v1/v1_apis_spec.rb

```diff
@@ -12,15 +12,15 @@
         expect(response).to have_http_status(200)
         expect(response.content_type).to eq('application/json')
         expect(response.charset).to eq('utf-8')
-        expect(response.body).to eq("{\"data\":{\"version\":\"#{version}\"}}")
+        expect(response.body).to eq("{\"data\":{\"version\":\"1.0\",\"providers\":{\"github.com\":{\"type\":\"github\",\"enabled\":true,\"id_application\":null,\"server\":\"api.github.com\",\"version\":\"v3\",\"verify\":true}}}}")
       end
 
-      it 'gets the API version' do
+      it 'gets the API version2' do
```

Comment:
?

Reply:
^^ It's hard to make an spec with -e '<name of the test>' when all the tests have the same name ^^I'll change it